### ISSUE TITLE
quiz solution

### ISSUE THREAD

**Issue by visusu** [stackup-dev/open-source-campaign#257]
**Name of Proposer**
visu

**Is your feature request related to a problem? Please describe.**
we want to learn also while earn

**Describe the solution you'd like**
give the solution of quiz so we can learn if we dont know the answer 

**Describe alternatives you've considered**
provide new quiz solution tab

**Additional context**
no image of solution available

**Comment by ishanvi525** [comment]
Name of Proposer
Ishanvi

Is your feature request related to a problem? Please describe.
Yes, whenever we're getting quizzes and not getting  rewarded for it. then at that moment it makes us feel frustrated. 

Describe the solution you'd like
**Answers to the quizzes must be provided after the deadlines are over. Since it would help us know where we faultered and would give us opportunity to correct us.
**I strongly believe that whenver the quests are not approved or rewarded, the must be told where they committed the error thus they can improve based on the feedback.

Describe alternatives you've considered
**Answers to the quizzes must be provided post facto deadline.
**the rejected quests must be given feedback why it was rejected.

Additional context
no image of solution available

**Comment by Thakur0005** [comment]
you should provide the answer of quiz after the submission of quest.So we can see our result

**Comment by ibukun-brain** [comment]
I agree with @visusu it would have been nice to have solutions to every quiz so that we can learn from our mistakes.

But as for the rejected quest he mentioned, we been told why in guides section, if we fail to meet up with the deliverable standards our quest submission will be rejected.
 
But I think it will be better too if we can have the solutions to the deliverable to see where we went wrong, this will definitely help a lot us as learning from mistakes or failure helps us to become a better versions of ourselves

**Comment by Tsebo-Knowledge** [comment]
I believe that the an addition of a document with correct answers that have a detailed brief explanation, should be attached to the quest feedback after the correction. In addition, discussions about misunderstood answers or concepts on the quest should be discussed on a given day. This discussions can be done in the StackUp server voice channels where a few members of the StackUp team could be a on the panel of discussion. These ideas can grant everyone with deep understanding of the mission.
